reference/library/src/morpheus-master/sass/_defaults.scss

```diff
@@ -34,8 +34,7 @@ $header-font-weight: 700 !default;										// 400 or 700 as you wish
 $background-color: 			 #FFFFFF !default;
 $background-color-secondary: #444444 !default;
 
-$primary-color:    			 #2a94c0 !default;
-$primary-color-shadow:			 #2A87C0 !default;
```

Comment:
You caught me @jonespm . I was trying to remove the overlapping look that the border provided. I'll remove it from my change.